blocks/inspector-controls/textarea-control/index.js

```diff
@@ -21,10 +21,9 @@ function TextareaControl( { label, value, help, instanceId, onChange, rows = 4,
 				rows={ rows }
 				onChange={ onChangeValue }
 				aria-describedby={ !! help ? id + '__help' : undefined }
+				value={ value }
```

Comment:
It looks like we could also remove it entirely as being injected by `{ ...props }` but I think keeping it explicit is good here (The value prop is required).